shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -65,6 +67,8 @@ - (instancetype)initWithProject:(FlutterDartProject*)projectOrNil
                          bundle:(NSBundle*)nibBundleOrNil {
   self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
   if (self) {
+    _weakFactory = std::unique_ptr<fml::WeakPtrFactory<FlutterViewController>>(
```

Comment:
Nit: `std::make_unique<fml::WeakPtrFactory<FlutterViewController>>(self)` is used in the engine for construction of unique pointer.